### ISSUE TITLE
Links in postings should have the attribute rel with the value "nofollow"

### ISSUE THREAD

**Issue by auge8472** [My-Little-Forum/mylittleforum#682]
Because of a new posting in the project forum with several links to the posters own pages I came across the absence of this addition to the links inside postings. The source code of the posting and also the code of [the corresponding function `make_link`](https://github.com/My-Little-Forum/mylittleforum/blob/f275e7395d0b7423564d4c4e59f9551d4be7eda3/includes/functions.inc.php#L406) made me aware of the absence of `rel="nofollow"` in the links. @loesler @ilosuna: I was of the opinion that this feature was already built in years ago. Am I wrong and remember this from another sofware/discussion (maybe MLF1 or SelfForum) or was it implemented and got lost somehow?

**Comment by loesler** [comment]
> I was of the opinion that this feature was already built in years ago. 

No. I don't remember such a discussion, and I personally don't like such a feature. I believe that Google is capable of dealing with such questionable links. So, from my knowledge, this was not implemented yet.

**Comment by auge8472** [comment]
> I believe that Google is capable of dealing with such questionable links.

Possibly they are but to be honest, marking a link with `rel="nofollow"` (and meanwhile additionally with `noopener` or `noreferrer`) to prevent a bot following it and to exclude it with it from the rank calculation was [a proposal of Google itself](https://en.wikipedia.org/wiki/Nofollow#Concept_and_specification).

Anyway, I seem to mix this with discussions in or more other platforms. It's at least implemented in the [Classic Forum](https://github.com/ckruse/cforum_ex), which is in use in the forum of SelfHTML.

**Comment by loesler** [comment]
> was [a proposal of Google itself
I know, but is was proposed in 2005.

> At least it would lower the urgency to act when a spam posting has passed the tests.
Sure, but it puts all users under general suspicion of linking questionable content. Just as cameras in public places will not prevent crime, the attribute will not prevent SPAM.
